seeker191-myupdate-now.md

```diff
@@ -0,0 +1 @@
+Hello! I am totally new to this.
```

Comment:
@seeker191 This looks great! Want to try a little markdown here? Maybe **bold** ?